index.js

```diff
@@ -47,7 +42,8 @@ Swarmerode.prototype.swarmInfo = function (cb) {
  * @param {Function} cb Callback with signature (err, hostExists).
  */
 Swarmerode.prototype.swarmHostExists = function (host, cb) {
-  this.swarmHosts(function (err, hosts) {
+  var consul = new Consul()
```

Comment:
This could return an error (as defined above). Do you need to try/catch here and pass the error to `cb`?